src/Core/Grid/Definition/Factory/LogGridDefinitionFactory.php

```diff
@@ -67,52 +76,60 @@ protected function getName()
+                (new BulkActionColumn('bulk_action'))
                 ->setOptions([
                     'bulk_value' => 'id_log',
                 ])
             )
-            ->add((new SimpleColumn('id_log'))
-                ->setName($this->trans('ID', [], 'Global.Actions'))
+            ->add(
+                (new SimpleColumn('id_log'))
+                ->setName($this->trans('ID', [], 'Admin.Global'))
             )
-            ->add((new EmployeeNameWithAvatarColumn('employee'))
+            ->add(
+                (new EmployeeNameWithAvatarColumn('employee'))
                 ->setName($this->trans('Employee', [], 'Admin.Global'))
             )
-            ->add((new SeverityLevelColumn('severity'))
+            ->add(
+                (new SeverityLevelColumn('severity'))
                 ->setName($this->trans('Severity (1-4)', [], 'Admin.Advparameters.Feature'))
                 ->setOptions([
                     'with_message' => true,
                 ])
             )
-            ->add((new SimpleColumn('message'))
+            ->add(
+                (new SimpleColumn('message'))
                 ->setName($this->trans('Message', [], 'Admin.Global'))
             )
-            ->add((new SimpleColumn('object_type'))
+            ->add(
+                (new SimpleColumn('object_type'))
                 ->setName($this->trans('Object type', [], 'Admin.Advparameters.Feature'))
             )
-            ->add((new SimpleColumn('object_id'))
+            ->add(
+                (new SimpleColumn('object_id'))
                 ->setName($this->trans('Object ID', [], 'Admin.Advparameters.Feature'))
             )
-            ->add((new SimpleColumn('error_code'))
+            ->add(
+                (new SimpleColumn('error_code'))
                 ->setName($this->trans('Error code', [], 'Admin.Advparameters.Feature'))
             )
-            ->add((new DateTimeColumn('date_add'))
+            ->add(
+                (new DateTimeColumn('date_add'))
                 ->setName($this->trans('Date', [], 'Admin.Global'))
                 ->setOptions([
                     'format' => 'Y-m-d H:i',
-                    'filter_type' => DateRangeType::class,
+                    'filter' => new ColumnFilterOption(DateRangeType::class),
                 ])
             )
-            ->add((new ActionColumn('actions'))
+            ->add(
+                (new ActionColumn('actions'))
                 ->setName($this->trans('Actions', [], 'Admin.Global'))
                 ->setOptions([
-                    'filter_type' => SubmitType::class,
-                    'filter_type_options' => [
-                        'label' => $this->trans('Search', [], 'Admin.Actions'),
+                    'filter' => new ColumnFilterOption(SearchAndResetFormType::class, [
                         'attr' => [
-                            'class' => 'btn btn-primary',
+                            'url' => $this->resetActionUrl,
```

Comment:
use `data-url` if you want to use custom attributes: https://developer.mozilla.org/en-US/docs/Learn/HTML/Howto/Use_data_attributes